javascript/modules/client/src/client-erc20.ts

```diff
@@ -163,46 +161,56 @@ export class ClientErc20 extends ClientCore implements IClientErc20 {
 
     const erc20VotingInstance = ERC20Voting__factory.connect(
       this._pluginAddress,
-      signer,
+      signer
     );
 
-    return erc20VotingInstance.estimateGas.newVote(
-      ...unwrapProposalParams(params),
-    ).then((gasLimit) => {
-      return this.web3.getApproximateGasFee(gasLimit.toBigInt());
-    });
+    return erc20VotingInstance.estimateGas
+      .newVote(...unwrapProposalParams(params))
+      .then(gasLimit => {
+        return this.web3.getApproximateGasFee(gasLimit.toBigInt());
+      });
   }
 
   private _estimateVoteProposal(proposalId: string, vote: VoteOptions) {
     // TODO: Unimplemented
-    return Promise.resolve(BigInt(0));
+    return Promise.resolve({ average: BigInt(0), max: BigInt(0) });
   }
   private _estimateExecuteProposal(proposalId: string) {
     // TODO: Unimplemented
-    return Promise.resolve(BigInt(0));
+    return Promise.resolve({ average: BigInt(0), max: BigInt(0) });
   }
   private _estimateSetDaoConfig(daoAddress: string, config: DaoConfig) {
     // TODO: Unimplemented
-    return Promise.resolve(BigInt(0));
+    return Promise.resolve({ average: BigInt(0), max: BigInt(0) });
   }
   private _estimateSetVotingConfig(daoAddress: string, config: VotingConfig) {
     // TODO: Unimplemented
-    return Promise.resolve(BigInt(0));
+    return Promise.resolve({ average: BigInt(0), max: BigInt(0) });
+  }
+
+  //// PRIVATE RETRIEVAL HANDLERS
+  private async _getDaoMembers(daoAddress: string) {
```

Comment:
If we're wrapping the result within a `Promise.resolve()`, there is not need for async yet.

```suggestion
  private _getDaoMembers(daoAddress: string) {
```

Reply:
Also, for consistency, I would expect `daoIdentifier` as stated on the readme (address / ENS)